Control/DeepSeq.hs

```diff
@@ -105,6 +114,10 @@ import GHC.Stack ( CallStack(..) )
 import GHC.SrcLoc ( SrcLoc(..) )
 #endif
 
+#if __GLASGOW_HASKELL__ >= 708
```

Comment:
Well, it's not something that personally bothers me that much, but I know @hvr has been wanting to be able to decouple `base` installations from GHC versions for a while, so I don't know if tying an conditional import from `base` to a GHC version will make things shaky in the future...